tests/integration-test-bq-update-acl.wdl

```diff
@@ -0,0 +1,30 @@
+version development
```

Comment:
Can this be part of https://github.com/susom/wdl-kit/blob/main/tests/integration-test-bq.wdl? So that it does the cleanup of created dtasets